src/symbols.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2020 Beyond Essential Systems Pty Ltd
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export function isOperator(token) {
+  return ['+', '-', '/', '*', 'x', 'u'].includes(token);
+}
+
+export function getPrecedence(operator) {
+  switch (operator) {
+    case 'u':
+      return 4;
+    case '*':
+    case 'x':
+    case '/':
+      return 3;
+    case '+':
+    case '-':
+      return 2;
+    default:
+      throw new Error('Invalid operator');
+  }
+}
+
+export function getOperands(formulaText) {
```

Comment:
Two questions about the specifications of this function:

1. Should it validate the formula first? (eg checking for umatched parentheses etc).
2. Should it parse numbers as integers in the result? Eg `getOperands('a + 1') // ['a', 1], currently: ['a', '1']`

I'd say yes for 1, and probably for 2 too

Now that I think of it, perhaps we should expose a validator function in the package's API

Reply:
I could go either way on both of those points, so I'm not much help!

Reply:
They are not critical at this stage so I'll leave them for another version